core/server/services/newsletters/service.js

```diff
@@ -185,6 +187,10 @@ class NewslettersService {
 
         const {cleanedAttrs, emailsToVerify} = await this.prepAttrsForEmailVerification(attrs, originalNewsletter);
 
+        if (originalNewsletter.status !== 'active' && cleanedAttrs.status === 'active') {
+            await this.limitService.errorIfWouldGoOverLimit('newsletters');
```

Comment:
I think this is missing passing `options.transacting` if it's set

Reply:
True, I didn't add it here because it wasn't used. But it is in fact. I'll also update the `findOne` call to use the transaction